### DISCUSSION TITLE
Using Block Editor in Independent Java Files

### DISCUSSION THREAD

**Discussion by kz3r01n** [Sketchware-Pro/Sketchware-Pro#662] [Ideas]
I'm just curious, but is it possible?

The reason why I'm asking is because having a lot of moreblocks to place the functions when java classes are a lot better to use, less lag, etc.

**Comment by j-px** [comment]
Technically it should be possible, but that'd require a lot of refactoring and new Blocks and maybe even a Component type to Sketchware Pro.
